examples/stack/values-minimal.yaml

```diff
@@ -0,0 +1,46 @@
+global:
+  observe:
+    collectionEndpoint: ""
+
+observe:
+  token:
+    value: ""
+
+events:
+  resources:
+    limits:
+      cpu: 50m
+      memory: 256Mi
+    requests:
+      cpu: 50m
+      memory: 256Mi
+
+logs:
+  fluent-bit:
+    resources:
+      limits:
+        cpu: 100m
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    config:
+      mem_buf_limit: 10MB
+
+metrics:
+  grafana-agent:
+    prom_config:
+    scrape_interval: 60s
+    scrape_configs:
+      cadvisor_action: drop
```

Comment:
I'm fine with this approach.

Why did we set `cadvisor_action: drop`?

https://observeinc.slack.com/archives/C06EBP24T1V/p1706044118395189

Reply:
Also the indentation is incorrect.

Reply:
Let's use this one including dropping `container_fs.*` as well.

```
metrics:
  grafana-agent:
    prom_config:
      scrape_interval: 60s
      scrape_configs:
        pod_action: drop
        cadvisor_metric_drop_regex: container_(network_tcp_usage_total|network_udp_usage_total|tasks_state|cpu_load_average_10s|fs_.*)
        cadvisor_metric_keep_regex: container_(cpu_.*|spec_.*|memory_.*|network_.*|file_descriptors)|machine_(cpu_cores|memory_bytes)
```

Reply:
BTW, we should make the same change for kustomize as well.

Reply:
This values-minimal.yaml is based on values-xs.yaml. What do we recommend if customers need to use a large or extra large templates?